docs/modules/ROOT/pages/advanced-how-tos/releasing/tenant-release-pipelines.adoc

```diff
@@ -27,44 +28,41 @@ NOTE: If you write a good reusable release pipeline, please submit it to our lin
 apiVersion: appstudio.redhat.com/v1alpha1
 kind: ReleasePlan
 metadata:
- labels:
-   release.appstudio.openshift.io/auto-release: 'true' <.>
-   release.appstudio.openshift.io/standing-attribution: 'true'
- name: publish <.>
- namespace: dev-workspace <.>
+  labels:
+    release.appstudio.openshift.io/auto-release: 'true'
+    release.appstudio.openshift.io/standing-attribution: 'true'
+  name: publish
+  namespace: dev-workspace
 spec:
- application: <application-name> <.>
- data:
-   mapping:
-     components: <.>
-       - name: demo-component-1
-         repository: registry/destination-image-repository-1
-         tags: [latest]
-       - name: demo-component-2
-         repository: registry/destination-image-repository-2
-         tags: [latest]
- tenantPipeline:
-   pipelineRef: <.>
-     resolver: git
-     params:
-       - name: url
-         value: "https://github.com/<your-github-user>/<your-pipeline-repo>.git"
-       - name: revision
-         value: main
-       - name: pathInRepo
-         value: "<path-to-your-pipeline>"
-   serviceAccountName: appstudio-pipeline <.>
+  application: <application-name>
+  data:
+    mapping:
+      components:
+        - name: demo-component-1
+          repository: registry/destination-image-repository-1
+          tags: [latest]
+        - name: demo-component-2
+          repository: registry/destination-image-repository-2
+          tags: [latest]
+  target: managed-workspace
+  tenantPipeline:
+    pipelineRef: <.>
+      resolver: git
+      params:
+        - name: url
+          value: "https://github.com/<your-github-user>/<your-pipeline-repo>.git"
+        - name: revision
+          value: main
+        - name: pathInRepo
+          value: "<path-to-your-pipeline>"
+    serviceAccountName: appstudio-pipeline <.>
 ----
 
 +
-<.> Optional: Control if Releases should be created automatically for this ReleasePlan when tests pass. Defaults to true.
```

Comment:
I think a lot of these notes were good to have

Reply:
They are in the linked document to define the ReleasePlan. I kept those relevant to the tenant pipeline.

Reply:
Sure, but today you can get all the details on this page, and now with this change you will have to go to another page and look at both at the same time to understand. Because it already has the notes here, I think they should be left here

Reply:
This wasn't in the documentation initially. It was added because there was user confusion.

Reply:
@davidmogar thoughts?